### DISCUSSION TITLE
Install finished but not work well on NanoPiR2s version v0.61 with kernel 6.6 or kernel 5.4.

### DISCUSSION THREAD

**Discussion by lusoe302** [Ysurac/openmptcprouter#3620] [General]
installed version is v0.60-6.1 then upgrade install via Flash new firmware image on web page method . 
It got installed and rebooted then it booted and just a little second we can call web config , then it got stack  , can't call config web anymore . 
we can still ping form WAN port IP 192.168.1.113 , can still ping 192.168.100.1 form LAN but It still loading webpage till forever .

Later Flash new firmware image with SD card write method with Balena Etcher , it booted but stack can't ping 192.168.100.1 from LAN , it also can't ping from WAN too(default config is not with DHCP) .

so I go back to v0.60-6.1 r0+24843-acf40c022e by SD card write method .

from v0.61 RC1 - v0.61 RC2 v0.61 - RC3 - OpenMPTCProuter v0.61 release 2024-10-23 06:27
same problem for all 0.61 .
32 GB SD card and Balena Etcher would be fine it work long with from 2023-09-25 to today .
so I go back to v0.60-6.1 it is fine with v0.60 .

**Comment by Ysurac** [comment]
Can you test latest snapshot ? ( https://snapshots.openmptcprouter.com/ )

**Reply by lusoe302** [reply]
same problem sir , all same , just a little second can load webpage then all same .

**Comment by lusoe302** [comment]
same problem sir , all same , just a little second can load webpage then all same .
